image_geometry/src/image_geometry/cameramodels.py

```diff
@@ -1,7 +1,4 @@
-import array
-
 import cv2
-import sensor_msgs.msg
```

Comment:
Just wondering, could removing these imports accidentally cause unintended breakages for users of this library? Unused dependencies would be good to remove, but probably only in the rolling branch (for ros2).

Reply:
Hmm, let me think about this. Maybe I'm getting this mixed up with #includes in cpp

Reply:
I'm completely wrong on this.